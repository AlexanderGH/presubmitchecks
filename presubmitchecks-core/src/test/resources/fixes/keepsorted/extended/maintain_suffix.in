Keeps comments with item but commas with position:
    // keep-sorted-test start maintain_suffix_order=(,)?(\s*(?://.*?|\/\*.*?|))
    C, // bob
    C, /* haha */
    1,
    D
      D, // :)
    B,
    B, // lol
    1,
    A
    // keep-sorted-test end
