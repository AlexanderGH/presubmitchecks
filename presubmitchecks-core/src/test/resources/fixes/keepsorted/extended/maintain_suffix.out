Keeps comments with item but commas with position:
    // keep-sorted-test start maintain_suffix_order=(,)?(\s*(?://.*?|\/\*.*?|))
    1,
    A,
    B,
    B, // lol
    C, /* haha */
    C, // bob
    D
      D // :)
    // keep-sorted-test end
